easybuild/easyconfigs/g/GMP/GMP-6.1.2-GCCcore-6.3.0.eb

```diff
@@ -21,6 +22,9 @@ builddependencies = [
 # enable C++ interface
 configopts = '--enable-cxx'
 
+# make sure that correct libgmp.so is picked up by tests when EasyBuild is configured with --rpath
+pretestopts = 'export LDFLAGS="-L%(builddir)s/.libs/ $LDFLAGS"'
```

Comment:
@boegel why didn't you add ` && ` at the end? I don't see how this could have worked without it

Reply:
It's basically just skipping the tests like this, so clearly not OK...

I'll fix this stupid mistake and re-test.

Reply:
fixed in 332cdb2